_search-plugins/cross-cluster-search.md

```diff
@@ -29,6 +29,12 @@ The following sequence describes the authentication flow when using cross-cluste
 1. The call, including the authenticated user, is forwarded to the remote cluster.
 1. The user's permissions are evaluated on the remote cluster.
 
+## Prerequisite
+
+In order to use proxy mode, you need to ensure that:
```

Comment:
Nit: If you plan to use proxy mode...

Reply:
This has now been changed to 
"To use proxy mode, fulfill the following prerequisites:"